packages/core/src/shipping/strategies/braintree/braintree-accelerated-checkout-shipping-strategy.ts

```diff
@@ -90,7 +86,7 @@ export default class BraintreeAcceleratedCheckoutShippingStrategy implements Shi
                 this._paymentMethodActionCreator.loadPaymentMethod(methodId),
             );
 
-            await this._runAuthenticationFlowOrThrow(methodId, styles);
+            await this._runAuthenticationFlowOrThrow(methodId, braintreefastlane?.styles);
```

Comment:
Is this async call only in case for fastlane or for all cases?

Reply:
For both Braintree Fastlane and Braintree Connect experiences. 
P.s.: Braintree Connect will be removed in the future